web/controllers/user_controller.ex

```diff
@@ -24,7 +24,9 @@ defmodule Org.UserController do
   end
 
   def show(conn, %{"id" => id}) do
-    user = Repo.get!(User, id)
+    user = User
+            |> Repo.get!(id)
```

Comment:
A lot of stuff I've seen seems pretty split. I started doing it that way I think because of one of the examples I'd seen, and just stuck with it to be consistent.